Dockerfile

```diff
@@ -1,3 +1,31 @@
+FROM public.ecr.aws/amazonlinux/amazonlinux:2 as builder
+
+# Install build dependencies for the package(s) below
+RUN \
+  yum -y install \
+    autoconf automake bison gettext-devel libtool make pkgconfig tar xz
+COPY ./sdk-fetch /usr/local/bin
+
+ARG utillinux_version=2.37.4
+
+WORKDIR /opt/build
+COPY ./hashes/util-linux ./hashes
+
+RUN \
+  sdk-fetch hashes && \
```

Comment:
Nit: since this isn't the SDK container, should `sdk-fetch` be just `fetch`, or `source-fetch`?

Reply:
It felt weird to rename it since the code itself is untouched between the SDK, admin container, and now the control container. I'm partial to `source-fetch` or `lookaside-fetch`, but if we change it I'd like to change it for all three repos.